Second training　Issue4~6修正

masa555555 opened the pull request:
what


why

masa555555 committed:
Issue4

masa555555 committed:
Issue4~6修正

norito0627 reviewed (commented):
レビュー依頼ありがとうございます！
こちらのプルリクエストにて、コンフリクトが起きていますので、まずはこちらを解消してください！
ブラウザエディタでコンフリクトを解消できますので、「Resolve conflicts」をクリックしましょう。

masa555555 committed:
Merge branch 'master' into second_training

masa555555 commented:
コンフリクト解消しました。

masayaomoto commented:
レビュー依頼ありがとうございます！
こちらですが、実際の画面や機能ができているか確認させていただきたいので、
キャプチャ動画をご提出いただけますか？
「Gyazo　使い方」とgoogleで検索し、ツールによって生成されたURLをこのPRに貼っていただければ幸いです。
お手数かけますが、よろしくお願いします。

masa555555 commented:
ブラウザ画面の表示です
https://i.gyazo.com/be944b1f350d1630239d8a631657ae65.jpg

masa555555 commented:
キャプチャー動画作成しました。
https://gyazo.com/915f6a024a5433e3d661962ccc08beb1

s-aka-moto commented on app/views/calendars/index.html.erb:
```diff
@@ -6,13 +6,19 @@
     <%= f.text_field :plan, class: 'text-input' %>
     <%= f.submit '保存' %>
   <% end %>
+ second_training
+</div>
+<div>
 
 
+
+master
```
不要な記述と思われますので、削除しておきましょう！
```
 second_training
</div>
<div>



master
```

s-aka-moto commented on app/controllers/calendars_controller.rb:
```diff
@@ -34,7 +35,14 @@ def get_week
+      if wday_num >= 7
+        wday_num = wday_num -7
+      end
+      days = { :month => (@todays_date + x).month, :date => (@todays_date+x).day, :plans => today_plans, :wday => wdays[wday_num] }
```
ハッシュロケットに戻ってしまっておりますので、シンボル型に修正しましょう！
`days = { :month => (@todays_date + x).month, :date => (@todays_date+x).day, :plans => today_plans, :wday => wdays[wday_num] }`

s-aka-moto reviewed (commented):
レビュー依頼ありがとうございます！
キャプチャ動画確認させていただきました！
コメントいたしましたので、ご確認をお願いいたします！

masa555555 committed:
修正しました

ハッシュロケットからシンボルに修正しました
days = { month:  (@todays_date + x).month, date:  (@todays_date+x).day, plans:  today_plans, wday:  wdays[wday_num] }

masa555555 committed:
削除しました

余分な記述を削除しました。

※修正前11行目に12行目にタグがありました
<div></div>

masa555555 committed:
削除しました

下記を削除しました

second_training
master

shukiyamamoto reviewed (commented):
レビュー依頼ありがとうございます。
LGTMとさせていただきます。issue4~6の実装お疲れさまでした。

マージ後、ローカルのmasterブランチに忘れずにプルしましょう。
引き続き実装頑張っていきましょう！
